src/components/Media/index.js

```diff
@@ -1,13 +1,41 @@
-import { StyledMedia, StyledBody, StyledItem } from './styled.js';
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import { space, flex } from 'styled-system';
 
-const Media = StyledMedia;
-Media.displayName = 'Media';
+import styles from './style.css';
+import withDefaultTheme from '../withDefaultTheme';
 
-const Body = StyledBody;
-Body.displayName = 'Media.Body';
+const FlexSpacer = withDefaultTheme(styled.div(flex, space));
```

Comment:
Is this going to have similar issues to what we were seeing before?

Reply:
No, this only provides the component props that would allow us to _set_ those values when we want. These don't add any styles by default